Created a auth folder containing the index and assets for the login page

vitthal-inani opened the pull request.

anuragiiits commented on auth/Login Page/index.html:
```diff
@@ -0,0 +1,140 @@
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/js/bootstrap.bundle.min.js"></script>
+</body>
+
+</html>
```
Add a new line at the end of every file.

Vineet-Sharma29 commented on auth/Login Page/index.html:
```diff
@@ -0,0 +1,140 @@
+            <div class="form-group"><input class="form-control" type="email" name="email" placeholder="Email"></div>
+            <div class="form-group"><input class="form-control" type="password" name="password" placeholder="Password"></div>
+            <div class="form-group"><button class="btn btn-primary btn-block" type="submit" style="background-color: rgb(25,0,86);">Log In</button></div><a class="forgot" href="#">Forgot your email or password?</a></form>
+    </div>
```
Avoid inline css. Replace class based css. Also try to use, bootstrap classes as much as possible. For example, for background color can use "bg" class

vitthal-inani commented:
![Screenshot Desktop](https://user-images.githubusercontent.com/44166227/68701092-10a53e00-05ac-11ea-9566-296234d24f6b.png)
![Screenshot Mobile](https://user-images.githubusercontent.com/44166227/68701112-1864e280-05ac-11ea-9b45-f39edcf5bb98.png)
Removed redundant css and added new lines
